node_helpers.py

```diff
@@ -22,3 +24,8 @@ def pillow(fn, arg):
         if prev_value is not None:
             ImageFile.LOAD_TRUNCATED_IMAGES = prev_value
         return x
+
+def hasher():
+    # This is a safe eval because args.default_hashing_function can ONLY 
+    # be one of four predefined strings, keeping it as a safe eval.
+   return eval(f"hashlib.{args.default_hashing_function}")
```

Comment:
Could you change this to just a switch? it's only 4 choices and `eval` is uncomfortable even if it's safe

Reply:
I have a nasty habit when we're pulling in / serving module level functions, etc. to other functions to use a dict to define acceptable value types.  This way I know exactly what is pulled in from where.

I have a hatred for long `if` chains and switches, so the changes in [e1de046](https://github.com/teward/ComfyUI/commit/e1de04604a7209acbbaa0e028f11fa8ef4b4575d) should make it safer.